spec/models/section_spec.rb

```diff
@@ -162,7 +163,7 @@ def key_classes_for(hash)
       end
 
       it 'builds a section using the editions' do
```

Comment:
I've reworded this example and added another to demonstrate the `Section` being built with both a previous and latest edition.